server/controllers/admin.py

```diff
@@ -352,14 +362,17 @@ def create_course():
 @is_staff(course_arg='cid')
 def course(cid):
     courses, current_course = get_courses(cid)
+
     students = (Enrollment.query
                           .options(db.joinedload('user'))
                           .filter(Enrollment.role == STUDENT_ROLE,
                                   Enrollment.course == current_course)
                             .all())
+    needs_intro = len(students) < 5
```

Comment:
This seems a little arbitrary. Not sure of a better condition, though. If the example assignment is the only assignment?